test/jdk/com/sun/jdi/SingleStepCompilationTest.java

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+//  THIS TEST IS LINE NUMBER SENSITIVE
+
+/**
+ * @test
+ * @bug 8229012
+ * @summary Verify that during single stepping a method is not compiled and
+ *          after single stepping it is compiled..
+ * @requires vm.compMode == "Xmixed"
+ * @library /test/lib /
```

Comment:
Q: Just want to make sure the`/` at the end is not a typo. :)

Reply:
Fixed. Probably I didn't fully delete a 2nd path that was copied from some other test.